### ISSUE TITLE
populate graphics defaults doesn't work on latest version

### ISSUE THREAD

**Issue by Baconfry** [Deijin27/RanseiLink#52]
Gives error message in 5.5. Users have to populate graphics defaults on an earlier version. 5.2 is confirmed to work

**Comment by Deijin27** [comment]
Yep I can't reproduce the error myself in the 5.5 release build. Can you send a screenshot of the error message please?

**Comment by Baconfry** [comment]
There was a user who had this issue where they couldn't populate graphics defaults, and I tested it myself and got the error message. After successfully populating graphics defaults on 5.2 I haven't been able to reproduce it either. I'll send a PSA
